@xen-orchestra/web/src/layouts/AppLayout.vue

```diff
@@ -1,8 +1,8 @@
 <template>
   <CoreLayout>
     <template #app-logo>
-      <RouterLink class="logo-link" to="/">
-        <LogoTextOnly :key="String(uiStore.isMobile)" :short="uiStore.isMobile" class="logo" />
+      <RouterLink class="logo-link" :class="{ mobile: uiStore.isMobile }" to="/">
+        <UiLogoText :text="t('xen-orchestra')" />
       </RouterLink>
```

Comment:
I think the xen-ochestra logo should go on the main dashoard

```suggestion
      <RouterLink class="logo-link" :class="{ mobile: uiStore.isMobile }" to="/dashboard">
        <UiLogoText :text="t('xen-orchestra')" />
      </RouterLink>
```